scripts/core/auth/auth-service.js

```diff
@@ -19,4 +19,25 @@ angular.module('superdesk.core.auth.auth', []).service('auth', ['$q', 'api', 'se
                         return session.identity;
                     }));
         };
+
+
+    /**
+     * Login using given credentials
+     *
+     * @param {string} jid (Jabber ID)
+     * @param {string} transaction ID
+     * @returns {object} promise
+     */
+        this.loginXmpp = function(jid, transactionId) {
+            function fetchIdentity(loginData) {
```

Comment:
imo there is no need for this wrapper function. I would just inline the command.